specification/operationsmanagement/resource-manager/Microsoft.OperationsManagement/2015-11-01-preview/examples/OperationsList.json

```diff
@@ -0,0 +1,50 @@
+{
+  "parameters": {
+    "api-version": "2015-11-01-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "name": "Microsoft.OperationsManagement/solutions/write",
+            "display": {
+              "provider": "Microsoft Operations Management",
+              "resource": "Solutions",
+              "operation": "Create new OMS solution",
+              "description": "Create new OMS solution"
+            }
+          },          
+          {
+            "name": "Microsoft.OperationsManagement/solutions/read",
+            "display": {
+              "provider": "Microsoft Operations Management",
+              "resource": "Solutions",
+              "operation": "Get existing OMS solution",
+              "description": "Get exiting OMS solution"
+            }
+          },
+          {
+            "name": "Microsoft.OperationsManagement/solutions/delete",
+            "display": {
+              "provider": "Microsoft Operations Management",
+              "resource": "Solutions",
+              "operation": "Delete existing OMS solution",
+              "description": "Delete existing OMS solution"
+            }
+          },
+          {
+            "name": "Microsoft.OperationsManagement/register/action",
+            "display": {
+              "provider": "Microsoft Operations Management",
+              "resource": "Register",
+              "operation": "Register a subscription to a resource provider.",
+              "description": "Register a subscription to a resource provider."
+            }
+          },
```

Comment:
There are JSON parse errors in the semantic and model validation, see the [travis](https://travis-ci.org/Azure/azure-rest-api-specs/builds/269677721?utm_source=github_status&utm_medium=notification) log.  While it's not clear from the error message I believe it's due to this trailing comma.